.github/workflows/package-dependencies-update.yml

```diff
@@ -0,0 +1,78 @@
+name: Update package dependencies versions
+
+# Set jobs to be configured and executed by schedule.
+on:
+  push:
+    branches:
+      - 'main'
```

Comment:
Can we run on release branches too?

Reply:
Yes, let me make those changes to the branches.

Reply:
Actually, we are still testing it, so let's start with just `main` for now.